app/src/main/report/components/Reports/ReportChangelog.tsx

```diff
@@ -22,11 +22,17 @@ export interface ReportChangelogProps extends ReportChangelogPublicProps {
     fetchChangelog: (props: ReportChangelogPublicProps) => void;
 }
 
-export const ReportChangelogComponent = (props: ReportChangelogProps) => {
+export class ReportChangelogComponent extends React.Component<ReportChangelogProps> {
+
+    render() {
+
+        if (this.props.versionChangelog == null) {
```

Comment:
Moved this from the recompose branch where it was previously, as this meant that componentDidUpdate was not called when it has branched to render LoadingElement instead. Not sure if this is the best approach though!